### ISSUE TITLE
multimatch prints the same directory for different matches

### ISSUE THREAD

**Issue by jtluka** [jpirko/lnst#164]
There's duplicate log message
```
Logs for this recipe will be stored in '/home/igyn/tmp/lnst/Logs/2016-01-30_11:12:31/ping_flood_match_1'
```
for both match 1 and match 2.

```
[igyn@dhcp-2-157 lnst (master)]$ ./lnst-ctl --multi-match -d match_setup ./recipes/regression_tests/phase1/ping_flood.xml 
Loading config file '/home/igyn/tmp/lnst/lnst-ctl.conf'
Loading config file '/home/igyn/.lnst/lnst-ctl.conf'
2016-01-30 11:12:31       (localhost)        -    INFO: 
    --------------------------------------------------------------------------------------------------------
    Trying recipe file "./recipes/regression_tests/phase1/ping_flood.xml" match 1
    Logs for this recipe will be stored in '/home/igyn/tmp/lnst/Logs/2016-01-30_11:12:31/ping_flood_match_1'
    --------------------------------------------------------------------------------------------------------
2016-01-30 11:12:31       (localhost)        -    INFO: No previous configuration found.
2016-01-30 11:12:31       (localhost)        -    INFO: Checking machine pool availability.
2016-01-30 11:12:31       (localhost)        -    INFO: Processing pool 'default', directory '/home/igyn/.lnst/pool'
2016-01-30 11:12:31       (localhost)        -   DEBUG: Querying machine 'lnst1': 192.168.122.220:9999
2016-01-30 11:12:31       (localhost)        -   DEBUG: Querying machine 'lnst2': 192.168.122.221:9999
2016-01-30 11:12:31       (localhost)        -    INFO: lnst1 [UP]    libvirt_domain: lnst1
2016-01-30 11:12:31       (localhost)        -    INFO: lnst2 [UP]    libvirt_domain: lnst2
2016-01-30 11:12:31       (localhost)        -    INFO: Finished loading pools.
2016-01-30 11:12:31       (localhost)        -   DEBUG: Executing: "cd "/home/igyn/tmp/lnst/test_tools" && tar cf "/tmp/tmpulP122" "multicast""
2016-01-30 11:12:31       (localhost)        -   DEBUG: Executing: "cd "/home/igyn/tmp/lnst/test_tools" && tar cf "/tmp/tmperigNS" "tcp_corrupt""
2016-01-30 11:12:31       (localhost)        -   DEBUG: Executing: "cd "/home/igyn/tmp/lnst/test_tools" && tar cf "/tmp/tmpA54D1Q" "tcp_conn""
2016-01-30 11:12:31       (localhost)        -    INFO: Matching machines, without virtuals.
2016-01-30 11:12:31       (localhost)        -    INFO: Trying match with pool: default
2016-01-30 11:12:31       (localhost)        -    INFO: Match failed for normal machines, falling back to matching virtual machines.
2016-01-30 11:12:31       (localhost)        -    INFO: Trying match with pool: default
2016-01-30 11:12:31       (localhost)        -    INFO: Pool match description:
2016-01-30 11:12:31       (localhost)        -    INFO:   Setup is using virtual machines.
2016-01-30 11:12:31       (localhost)        -    INFO:   host "machine1" uses "lnst1"
2016-01-30 11:12:31       (localhost)        -    INFO:   host "machine2" uses "lnst2"
2016-01-30 11:12:31       (localhost)        -    INFO: Matching machines, without virtuals.
2016-01-30 11:12:31       (localhost)        -    INFO: Trying match with pool: default
2016-01-30 11:12:31       (localhost)        -    INFO: 
    --------------------------------------------------------------------------------------------------------
    Trying recipe file "./recipes/regression_tests/phase1/ping_flood.xml" match 2
    Logs for this recipe will be stored in '/home/igyn/tmp/lnst/Logs/2016-01-30_11:12:31/ping_flood_match_1'
    --------------------------------------------------------------------------------------------------------
2016-01-30 11:12:31       (localhost)        -    INFO: Pool match description:
2016-01-30 11:12:31       (localhost)        -    INFO:   Setup is using virtual machines.
2016-01-30 11:12:31       (localhost)        -    INFO:   host "machine1" uses "lnst2"
2016-01-30 11:12:31       (localhost)        -    INFO:   host "machine2" uses "lnst1"
2016-01-30 11:12:31       (localhost)        -    INFO: Matching machines, without virtuals.
2016-01-30 11:12:31       (localhost)        -    INFO: Trying match with pool: default
2016-01-30 11:12:31       (localhost)        - WARNING: Match 3 not possible.
2016-01-30 11:12:31       (localhost)        -    INFO: ============================ SUMMARY ============================
2016-01-30 11:12:31       (localhost)        -    INFO:  ./recipes/regression_tests/phase1/ping_flood.xml match: 1  PASS 
2016-01-30 11:12:31       (localhost)        -    INFO:      Pool match description:  
2016-01-30 11:12:31       (localhost)        -    INFO:      Setup is using virtual machines.  
2016-01-30 11:12:31       (localhost)        -    INFO:      host "machine1" uses "lnst1"  
2016-01-30 11:12:31       (localhost)        -    INFO:      host "machine2" uses "lnst2"  
2016-01-30 11:12:31       (localhost)        -    INFO:  ./recipes/regression_tests/phase1/ping_flood.xml match: 2  PASS 
2016-01-30 11:12:31       (localhost)        -    INFO:      Pool match description:  
2016-01-30 11:12:31       (localhost)        -    INFO:      Setup is using virtual machines.  
2016-01-30 11:12:31       (localhost)        -    INFO:      host "machine1" uses "lnst2"  
2016-01-30 11:12:31       (localhost)        -    INFO:      host "machine2" uses "lnst1"  
2016-01-30 11:12:31       (localhost)        -    INFO: =================================================================
2016-01-30 11:12:31       (localhost)        -    INFO: Total test time: 0 days, 0 hours, 0 minutes, 0 seconds
2016-01-30 11:12:31       (localhost)        -    INFO: Logs are stored in '/home/igyn/tmp/lnst/Logs/2016-01-30_11:12:31'
```

However, listing of directory shows:
```
[igyn@dhcp-2-157 lnst (master)]$ ls -1 /home/igyn/tmp/lnst/Logs/2016-01-30_11:12:31
debug
info
ping_flood_match_1
ping_flood_match_2
```

**Comment by jtluka** [comment]
Patch sent: https://lists.fedorahosted.org/archives/list/lnst-developers@lists.fedorahosted.org/thread/Y6O6GNDO77YZVNMYFE3XNGTFZNIRRCVG/

**Comment by jtluka** [comment]
Fixed by commit 6eb9665898b6aa2860b90eef39f245299fb23941
